app/services/emoji_converter.rb

```diff
@@ -1,15 +1,7 @@
 class EmojiConverter
```

Comment:
I think we should change the name of the service to an action, see https://docs.dev.to/backend/service-objects/

Something like `Html::ParseEmoji`, this way we can also move `CodeBlockParser` to something lik `Html::ParseCodeBlock` 

What do you think?

Reply:
Great idea - updated!